examples/ng_networking/Makefile

```diff
@@ -0,0 +1,35 @@
+# name of your application
+APPLICATION = ng_networking
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Include packages that pull up and auto-init the link layer.
+# NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
+USEMODULE += ng_netif
+USEMODULE += auto_init_ng_netif
+# Specify the mandatory networking modules for IPv6 and UDP
+USEMODULE += ng_ipv6_default
+USEMODULE += ng_udp
+# This application dumps received packets to STIO using the pktdump module
+USEMODULE += ng_pktdump
+# Additional networking modules that can be dropped if not needed
```

Comment:
I know what this line means, but with my newbie glasses my only thought is "Huh? Which additional networking modules? Who can drop what?"

I think the simplest and clearest comment would just be

```Makefile
USEMODULE += ng_icmpv6_echo   # (optional, remove if ICMPv6 ping functionality is not wanted)
```